src/app/components/message/message.component.ts

```diff
@@ -9,14 +10,20 @@ import { MessageServices } from '../../services/message.service';
 })
 export class MessageComponent  {
   messages: Message[];
+  moreMessages: boolean;
   constructor(private messageService: MessageServices){
     this.messageService.getMessages().subscribe(messages => {
       this.messages = messages;
     });
+    this.moreMessages = false;
   }
 
-  reading() {
-
+  showMore(){
+    if(this.moreMessages === true){
```

Comment:
you can have only this.messages if(this.moreMessages). I would also suggest to rename moreMessages to isMoreMessage.